llvm/test/CodeGen/AMDGPU/copy_phys_vgpr1024.mir

```diff
@@ -0,0 +1,30 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+# RUN: llc -mtriple=amdgcn -mcpu=gfx90a -run-pass postrapseudos -verify-machineinstrs %s -o - | FileCheck -check-prefix=GFX90A %s
+
+---
+name: copy_v1024_to_v1024_fwd
+tracksRegLiveness: true
+body: |
+  bb.0:
+    liveins: $vgpr2_vgpr3_vgpr4_vgpr5, $vgpr32
+    ; GFX90A-LABEL: name: copy_v1024_to_v1024_fwd
+    ; GFX90A: liveins: $vgpr2_vgpr3_vgpr4_vgpr5, $vgpr32
+    ; GFX90A-NEXT: {{  $}}
+    ; GFX90A-NEXT: $vgpr0_vgpr1 = V_PK_MOV_B32 8, $vgpr32_vgpr33, 12, $vgpr32_vgpr33, 0, 0, 0, 0, 0, implicit $exec, implicit $vgpr32_vgpr33_vgpr34_vgpr35_vgpr36_vgpr37_vgpr38_vgpr39_vgpr40_vgpr41_vgpr42_vgpr43_vgpr44_vgpr45_vgpr46_vgpr47_vgpr48_vgpr49_vgpr50_vgpr51_vgpr52_vgpr53_vgpr54_vgpr55_vgpr56_vgpr57_vgpr58_vgpr59_vgpr60_vgpr61_vgpr62_vgpr63, implicit-def $vgpr0_vgpr1_vgpr2_vgpr3_vgpr4_vgpr5_vgpr6_vgpr7_vgpr8_vgpr9_vgpr10_vgpr11_vgpr12_vgpr13_vgpr14_vgpr15_vgpr16_vgpr17_vgpr18_vgpr19_vgpr20_vgpr21_vgpr22_vgpr23_vgpr24_vgpr25_vgpr26_vgpr27_vgpr28_vgpr29_vgpr30_vgpr31
```

Comment:
While expanding `V[0:31] = COPY V[32:63]`, Src tuple is marked as `implicit` (for every pair, not just for the first) and Dst tuple is marked as `implicit-def` (in the first and last).
Everything looks fine here or do we also need to mark Src tuple as `implicit-def` ?

Reply:
The last instruction is picking up an extra exec use for some reason

Reply:
But, we don't have similar bug here right ?